constructor/construct.py

```diff
@@ -42,7 +42,7 @@
 '''),
 
     ('channels_remap',         False, list, '''
-A list of `src/dest` channel pairs. When building the installer, conda will
+A list of `src/dest` channel URL pairs. When building the installer, conda will
```

Comment:
```suggestion
A list of `src/dest` channel/URL pairs. When building the installer, conda will
```

Reply:
That makes it look like it's either a channel (name) or a URL, but I want to say "only URLs pointing to a channel". Is that clear or should I rephrase?

Reply:
Ah, I saw it as `src/dest` becomes `channel/URL`. Funny how different perceptions are. It's certainly not a blocker, just a suggestion.